src/app/screens/SignUp/styles.module.scss

```diff
@@ -12,10 +12,10 @@
     background-color: $cerulean;
   }
 
-  .wolox-logo-wrapper {
+  .logo-container {
```

Comment:
why do you need a div contaainer for the image aand separate clases for image and container?